style.css

```diff
@@ -19,8 +16,6 @@
 .introduction {
   width: 375px;
   height: 720px;
-
-  /* margin: 0 auto; */
```

Comment:
[OPTIONAL]

- [ ] Please don't use any fixed widths, your web app should be responsive for all mobile devices this will be covered in future lessons but for now, you should not use any fixed widths. 😁